knime.yml

```diff
@@ -0,0 +1,22 @@
+group_id: se.redfield
+name: bert
+author: Redfield AB
+vendor: Redfield AB
+description: "Redfield BERT Nodes"
+long_description: |
+  The extension provides the nodes that allow to use Google BERT Python libarary without coding.
```

Comment:
Suggestion:
This extension provides nodes that make the functionality of Google's BERT library available inside KNIME.